Dockerfile

```diff
@@ -35,8 +35,8 @@ ENV CEPH_DEVEL false
 ENV CEPH_REF quincy
 ENV OSD_FLAVOR default
 
-# Optional custom .deb repo
-ARG CUSTOM_APT_REPO=""
+# Additional custom .deb repo
+ARG CUSTOM_APT_REPO="ppa:peter-sabaini/ceph-test"
```

Comment:
I'd much rather this be an optional config, and passed in via configuration as with the version on the main branch

Reply:
The problem  is that if you do not provide "CUSTOM_APT_REPO" build argument while building the docker image the process will fail since cephfs-mirror is a mandatory package and has no official apt repo to provide it. Thus this does not stay "optional" rather mandatory. I am particularly hesitant to add the build argument in the script because then there will be 1 more arbitrary place (script rather than Dockerfile) where this will have to be removed once the package is officially available.

Reply:
That's true, today, but the changes in that PPA must merge into the ceph package in Ubuntu; having it present, _at all_, is merely a bandaid so we can move forward; the deeper we embed that PPA, the harder it is to deal with later when we're ready to remove it :-/

Reply:
hmm, that makes sense. For this I will: 
1. Modify our readme (with a note) to specify that we need to pass in an additional build-arg for the image to build without failure.
2. (In script Test) Use similar gh workflow injected build-arg to the script as well that way our code will not contain hardcoded references to the apt repo (workflows will).

Any suggestions regarding this ?

Reply:
The image build shouldn't fail either way? I would be cautious encoding "you need to use this PPA when building" in documentation, given that docs are very sticky - and what's the point of having a script that builds the docker images for local testing if we then expect people to build another way that would require them to understand this PPA?

Reply:
```docker build -t canonical/ceph:latest .```
will fail unless ```--build-arg CUSTOM_APT_REPO="ppa:peter-sabaini/ceph-test"``` is additionally provided. The workflows have that additional build arg hence tests work perfectly well. My intent being a simple build should not fail or we should document how to build the image without failure (for band-aid situations).